src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,20 @@
+import Logo from './Logo'
+import { UserCircle } from 'lucide-react'
+import { Link } from 'react-router-dom'
+
+export default function Navbar() {
+  const handleUserClick = () => {
+    console.log('Clicked UserIcon!')
+  }
+
+  return (
+    <nav className="w-full border-b border-gray-300">
```

Comment:
Instead of a border I would suggest that we use a background color for the Navbar to make it more distinguishable from the rest of the site.